pep/app/pods/components/home/component.ts

```diff
@@ -39,22 +42,42 @@ export default class Home extends Component<HomeArgs> {
      */
     graphicWrapWidth = 725;
 
+    cookieName = 'pep_last_viewed_expert_pick';
+
     @tracked model?: Abstract;
     @tracked imageArticle?: SearchDocument;
+    @tracked currentlyShownIndex: number = 0;
 
     get intro() {
         return this.configuration.content.home.intro;
     }
 
     get expertPick() {
         const expertPicks = this.configuration.base.home.expertPicks;
-        return expertPicks[expertPicks.length - 1];
+        return expertPicks[this.currentlyShownIndex];
     }
 
     get imageArticleUrl() {
         return this.imageArticle?.id ? this.router.urlFor('browse.read', this.imageArticle?.id) : '';
     }
 
+    /**
+     * Save expert pick viewed index and date to a cookie. We need to do this to figure out what they viewed, and when to switch them to the next item in the list
+     *
+     * @param {number} index
+     * @memberof Home
+     */
+    saveExpertPickViewToCookie(index: number): void {
+        this.cookies.write(
+            this.cookieName,
+            JSON.stringify({
+                index,
+                dateViewed: new Date()
+            }),
+            {}
+        );
+    }
```

Comment:
this implementation (using cookies to determine which expert pick to show) seems like it might be at odds with what they really want here, as this means different users (and even the same user on different browsers/devices) will potentially see a different "pick of the day", depending on when they first viewed, right?

@adistasio can correct me if I'm wrong but I assumed they'd want the "pick of the day" to be the same for everyone on any given day? We could actually do this pretty easily, and w/alot less complexity/no cookies, if we just added another config, `home.expertPicksStartDate` where they enter a date value. Then on render of the page you simply get the difference between that date and the current date, and that's your index of which expert pick to show in the list (and if its greater than the list length, you just loop back around).

Reply:
@billdami yes, do the start date